### PR TITLE
JBIDE-23303 - NullPointerException in ELValidator.validateElOperand

### PR DESCRIPTION
**Pull request by vrubezhny**
NPE is prevented.

Signed-off-by: Victor Rubezhny <vrubezhny@redhat.com>

### CODE REVIEW

File: plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java [hunk 1]
```diff
@@ -510,7 +510,7 @@ private void validateElOperand(ELReference elReference, ELInvocationExpression o
 		if(token==null) {
 			WebKbPlugin.getDefault().logError("The token from unresolved segment is null. EL: [" + operand + "]");
 		}
-		varName = token.getText();
+		varName = token != null ? token.getText() : null;
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java]: We should probably wrap the rest of the method in } else { ...} block instead. See lines 510-512.

Comment (vrubezhny) [inline plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java]: Actually. I believe that adding "return null;" in the end of "if(token==null){}" body (after the error message is logged) would be better as we don't need to worry of the rest operations in this case (and actually not able to do anything more).

**Comment by vrubezhny** [comment]
I've pushed an updated fix that just adds 'return;' statement after the error logging. I don't see a need of moving that amount of the rest code into any blocks as we actually cannot do anything else after token is found to be null.

File: plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java (hunk 1 above)
Comment (alexeykazakov) [inline plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java]: It's matter of your personal preferences. The logic will be the same. So feel free to use "return".

**Pull request merged**